src/vs/workbench/browser/parts/statusbar/statusbarPart.ts

```diff
@@ -908,9 +934,14 @@ class StatusbarEntryItem extends Disposable {
 			this.labelContainer.setAttribute('role', entry.role || 'button');
 		}
 
-		// Update: Hover (on the container, because label can be disabled)
+		// Update: Hover
 		if (!this.entry || !this.isEqualTooltip(this.entry, entry)) {
-			this.hover.value = setupCustomHover(this.hoverDelegate, this.labelContainer, { markdown: entry.tooltip, markdownNotSupportedFallback: undefined });
+			const hover = { markdown: entry.tooltip, markdownNotSupportedFallback: undefined };
+			if (this.hover.value) {
+				this.hover.value.update(hover);
+			} else {
+				this.hover.value = setupCustomHover(this.hoverDelegate, this.container, hover);
```

Comment:
I do not know why we chose to install the hover on `this.labelContainer` and not `this.container` because the hover should imho appear over the entire entry, not just the label?

Reply:
I don't know the history either. I just kept the status quo